### DISCUSSION TITLE
Inventory Kit Documentation

### DISCUSSION THREAD

**Discussion by gavinbc** [medusajs/medusa#10420] [Q&A] [answered]
Exploring MedusaJS as a possible alternative to Woo and Shopify, so far pretty impressed. Long time Woocommerce/Shopify developer, and I'm a bit of late comer to the jamstack way of doing things, but absolutely fed up with being tied to a monolithic platform. Started off exploring headless Woocommerce, ended up looking heavily into 11ty, few other rabbit holes, and here I am.

Right off the bat, one of the issues I need to resolve is how to handle product bundles (in Shopify/Woocommerce), which as far as I can tell is called Inventory Kits in Medusa. That said, I'm having no luck finding any documentation for said functionality.  It also feels somewhat half baked as currently implemented and a bit buggy. Although that could very well be related to my lack of understanding of the process.

So, documentation for Inventory Kits, does it exist somewhere? Either way thanks for the help!

**Comment by shahednasser** [comment]
Thank you for sharing this feedback! We currently don't have documentation around it but have it on our todo list. You can use inventory kits in the admin dashboard to build bundles. Let's say you have a bundle of a shirt and pants, you can create an inventory item for the shirt and an inventory item for the pant, then create a product whose variant's inventory kit points to the inventory items for the shirt and pant.

Let me know if this is clear. I'm also happy to ping you when we have the documentation around it too.

**Reply by SalahAdDin** [reply]
The Gift Cards module, will it be for pay?

**Comment by shahednasser** [comment, marked as answer]
Hi @gavinbc we've documented a conceptual guide around inventory kits here: https://docs.medusajs.com/resources/commerce-modules/inventory/inventory-kit

**Reply by squadcar** [reply]
Thanks @shahednasser, definitely looks like it works as expected conceptually. My issue with the process being buggy/incomplete is still a concern, so I'll make a separate post about it. Thanks again though for the follow up :)
